python/tools/create_inbox_message/create_inbox_message.py

```diff
@@ -0,0 +1,33 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2024 ftrack
+
+import ftrack_api
+
+session = ftrack_api.Session()
+
+recipients_user_names = ['<username>']
+message_content = 'Message body'
```

Comment:
@seebergs Change this to something like "Example Message Content from recipe" so is easy to spot.